packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -105,29 +113,35 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   }
 
   private _getDescriptiveOverflowElement(numPersonasToShow: number): JSX.Element {
+    let { personaSize } = this.props;
     let numPersonasNotPictured: number = this.props.personas.length - numPersonasToShow;
 
     if (!this.props.overflowButtonProps || numPersonasNotPictured < 1) { return null; }
 
     let personaNames: string = this.props.personas.slice(numPersonasToShow).map((p: IFacepilePersona) => p.personaName).join(', ');
 
     return <button { ...getNativeProps(this.props.overflowButtonProps, buttonProperties) }
-      className={ css('ms-Facepile-descriptiveOverflowButton', 'ms-Facepile-itemButton') }
-      title={ personaNames }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-descriptiveOverflowButton', 'ms-Facepile-itemButton') }
+      title={ personaNames }
+      >
       { '+' + numPersonasNotPictured }
     </button>;
   }
 
   private _getIconElement(icon: string): JSX.Element {
+    let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.overflowButtonProps, buttonProperties) }
-      className={ css('ms-Facepile-overflowButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-overflowButton', 'ms-Facepile-itemButton') }
+      >
       <i className={ css('ms-Icon', 'msIcon', `ms-Icon ms-Icon--${icon}`) } aria-hidden='true'></i>
     </button>;
   }
 
   private _getAddNewElement(): JSX.Element {
+    let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.addButtonProps, buttonProperties) }
-      className={ css('ms-Facepile-addButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton') }
+      tabIndex={ 0 }>
```

Comment:
I originally had tabIndex={ 1 } trying to get the tab order to change.  I want the focusZone first and then the Add button.